### ISSUE TITLE
Broken demo

### ISSUE THREAD

**Issue by khylias** [jonasmedeiros/ng-sticky#12]
Hi,

just a little issue to signal that the demo on plunkr is broken.

**Comment by jonasmedeiros** [comment]
Yes I saw the Angular CDN was removed I will check to bring a new example the library
